common/test/data/aside/html/toyhtml.xml

```diff
@@ -1 +1 @@
-<html filename="toyhtml.html"/>
\ No newline at end of file
+<html filename="toyhtml"/>
```

Comment:
Is this change going to break exports, imports, or any university's random in-house OLX parsing scripts?

Reply:
This change to remove support for `filename` having `.html` in the name is in a separate commit that that we can skip

@ormsbee Should we keep this backwards compatibility feature?

Reply:
I'm not sure I understand the full implications. Does this mean that the code is always going to look for `toyhtml.html` when it sees `filename="toyhtml"`? What happens if a unit has this bit of OLX in it? Will it prefer `toyhtml.xml` or `toyhtml.html`?

I think that it's fine to remove some of the more esoteric backwards compatibility matching (e.g. `.html.html` and `.html.xml` extensions), because they were frankly bugs that we needed to work around in 2012. But I think it's worth allowing someone to explicitly keep the `.html` extension here.

Reply:
With these changes, for any `filename="X"`, it would look only for the file `X.html`.

If we do want to keep the option for authors specifying `filename="f.html"` where the file will `f.html` or filenames can end in `.xml` etc, the only code I think we can remove is the first two ifs in this method:
```
   @classmethod
    def backcompat_paths(cls, filepath):
        """	            
        Get paths for html and xml files.
        """
        if filepath.endswith('.html.xml'):	    }
            filepath = filepath[:-9] + '.html'  # backcompat--look for html instead of xml	
        if filepath.endswith('.html.html'):	
            filepath = filepath[:-5]  # some people like to include .html in filenames..
```

Does that sound good or should I leave all the `backcompat` changes as is (they are all in the second commit).

Reply:
Since there are not a lot of tests which cover these cases, in the interest of reducing risk, I have undone the changes related to this for now.